core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2199,6 +2199,9 @@ public AdministrativeMonitor getAdministrativeMonitor(String id) {
      * @since 2.64
      */
     public List<AdministrativeMonitor> getActiveAdministrativeMonitors() {
+        if (!Jenkins.get().hasPermission(ADMINISTER)) {
+            return Collections.emptyList();
+        }
```

Comment:
Hide administrative monitors so they won't show up on `/manage`.

Reply:
I would argue that Jenkins "managers" may want to see some administrative warnings in the future, but it is a subject for future enhancements

Reply:
These are difficult due to how they're secured, and have been secured, for a long time.

Similar to all other `/manage` entries besides the trivial ones whose URLs are accessible to anyone anyway, I consider this a topic for future improvements that can be reviewed and debated independently.